site/en/docs/webstore/manage/index.md

```diff
@@ -119,9 +119,10 @@ Users will be able to communicate with you using the **Support** tab of your sto
 
 ### Using the User Support tab {: #user-support-tab }
 
-First, you must enable user feedback in the developer console Account setting page.
+Having a Support section on your extension's Chrome Web Store page allows users to easily report issues and suggest new features separately from leaving a review. To enable user feedback on your extension, the Support Visibility option must be switched on.   
+The Support Visibility option switch is located in the Chrome Web Store Developer Dashboard, in the Support Tab section of Account Options.
```

Comment:
ATM a single line break causes the text to render with a `<br>` inserted between the lines. We should either remove the line break to make this a single paragraph or add another line break to make "The Support Visibility option…" a distinct paragraph.

```suggestion
Having a Support section on your extension's Chrome Web Store page allows users to easily report issues and suggest new features separately from leaving a review. To enable user feedback on your extension, the Support Visibility option must be switched on.

The Support Visibility option switch is located in the Chrome Web Store Developer Dashboard, in the Support Tab section of Account Options.
```

**Current rendering**

![Screen Shot 2022-11-07 at 12 42 32 PM](https://user-images.githubusercontent.com/1047763/200412460-77e23778-b3b6-44cb-a4e0-e05e1a9ff7c2.png)